adr/00004-encrypt-database-fields.md

```diff
@@ -0,0 +1,31 @@
+# 8. Encrypt database fields
+
+Date: 2023-08-17
+
+## Status
+
+Accepted
+
+## Context
+
+Currently, the database isn't encrypted at the application level and any leak of data
```

Comment:
I don't think this is quite right... "Currently the database is encrypted at rest but unauthorised access to a running database instance could still result in exposure of PII"